src/main/java/thePackmaster/actions/pixiepack/EnchantmentAction.java

```diff
@@ -0,0 +1,44 @@
+package thePackmaster.actions.pixiepack;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.utility.NewQueueCardAction;
+import com.megacrit.cardcrawl.actions.utility.UnlimboAction;
+import com.megacrit.cardcrawl.actions.utility.WaitAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+
+public class EnchantmentAction extends AbstractGameAction {
+
+    AbstractCard toPlay;
+    public EnchantmentAction(AbstractCard card, AbstractCreature newTarget)
+    {
+        this.actionType = ActionType.CARD_MANIPULATION;// 21
+        this.duration = this.startDuration = Settings.ACTION_DUR_FAST;
+        toPlay = card;
+        target = newTarget;
+    }
+    @Override
+    public void update() {
```

Comment:
Have you tested whether Enchantments trigger things like Beat of Death or Gremlin Nob's enrage? With how this code is going through all the card playing logic, it might result in things like Beat of Death being triggered, and I'm not sure if you want that or not.

If you test it out and this is working as expected that's completely fine.

Reply:
They do indeed which is consistent with their current wording which reads:
"When you play a card with a different color any applicable Enchantments are copied and played onto the same target."